user_docs/en/userGuide.md

```diff
@@ -93,8 +93,13 @@ For details regarding exceptions, access the license document from the NVDA menu
 
 ## NVDA Quick Start Guide {#NVDAQuickStartGuide}
 
-This quick start guide contains three main sections: downloading, initial setup, and running NVDA.
-These are followed by information on adjusting preferences, using add-ons, participating in the community and getting help.
+This quick start guide contains three main sections:
```

Comment:
Should we make this caps for branding? Like we do for "User Guide" and "Changes"

```suggestion
This Quick Start Guide contains three main sections:
```